libraries/plugins/elasticsearch/elasticsearch_plugin.cpp

```diff
@@ -227,23 +227,37 @@ void elasticsearch_plugin_impl::doBlock(uint32_t trx_in_block, const signed_bloc
 
 void elasticsearch_plugin_impl::doVisitor(const optional <operation_history_object>& oho)
 {
+   graphene::chain::database& db = database();
+
    operation_visitor o_v;
    oho->op.visit(o_v);
 
    vs.fee_data.asset = o_v.fee_asset;
+   vs.fee_data.asset_name = o_v.fee_asset(db).symbol;
    vs.fee_data.amount = o_v.fee_amount;
+   vs.fee_data.amount_calculated = (float)(o_v.fee_amount.value)/pow(10, o_v.fee_asset(db).precision);
```

Comment:
``precision`` is not greater than 12, can use a lookup table instead of expensive ``pow`` operation.
IMO ``amount_calculated`` is a bad name, perhaps better ``amount_units``?
Using float is not nice but ok in this context I guess. Perhaps better use double?

(Same for the other ``_calculated`` fields of course.)